tt_metal/impl/CMakeLists.txt

```
@@ -45,6 +45,18 @@ set(IMPL_SRC
     ${CMAKE_CURRENT_SOURCE_DIR}/event/event.cpp
 )
 
+# Include helper functions and generate headers from flatbuffer schemas
+include(${PROJECT_SOURCE_DIR}/cmake/flatbuffers.cmake)
```

```suggestion
include(flatbuffers)
```

The CMAKE_PATH has been configured for the project.  A consumer shouldn't have intimate knowledge of the infra organization.  They're two different concerns.

---

thanks, this works.